docs/src/modules/branding/CommunitySayCard.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Box, Avatar, Typography } from '@material-ui/core';
+
+interface CommunitySayCardProps {
+  name: string;
+  id: string;
+  description: any;
+  avatar: string;
+  uniqueKey: number;
+}
+export default function CommunitySayCard(props: CommunitySayCardProps) {
+  const { name, id, description, avatar, uniqueKey } = props;
+  return (
+    <Box
+      sx={{
+        bgcolor: 'greyF3',
+        px: { xs: 4, lg: 5 },
+        py: 5,
+        pt: 5.8,
+        mt: uniqueKey === 1 ? { sm: 0, lg: 3.3 } : 0,
+      }}
+    >
+      <Box
+        component="img"
+        src="/static/branding/mui-x/twitter.svg"
+        loading="lazy"
+        alt="Twitter"
+        sx={{
+          mb: 3.8,
+        }}
+      />
+      <Typography variant="h4" align="center" sx={{ mb: { xs: 4, lg: 3 }, textAlign: 'left' }}>
+        {description}
+      </Typography>
+      <Box sx={{ display: 'flex', alignItems: 'center' }}>
+        <Avatar
+          sx={{
+            mr: { xs: 1.5, lg: 2 },
+            bgcolor: 'sunglow',
```

Comment:
Maybe a grey from the theme would work better

Reply:
That color is not needed anymore. I found the correct avatars and assigned them diractly.

Reply:
The background color is important for the loading state, my point was that it seems to be outside of the palette.